src/views/Generator/TestOptions/VariablesEditor.tsx

```diff
@@ -54,13 +54,15 @@ export function VariablesEditor() {
   return (
     <form onSubmit={handleSubmit(onSubmit)}>
       <Text size="2" as="p" mb="2">
-        Add variables to be used in your test script.
+        Define custom variables and use them in your custom code rules, for
+        example:
+        <Code>{'VARS["variable_0"]'}</Code>.
```

Comment:
We may want to remove punctuation in the second part, because `:VARS["variable_0"].` looks a bit off